### PR TITLE
Explore Page

### LINKED ISSUE #7: Explore Page
- Should list all the videos available
- Should be able to filter videos based on category

### PR DESCRIPTION
**Pull request by ttahm3d**
- Lists all the videos
- Filters the videos based on the selection

### CODE REVIEW

File: src/context/videos/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { VideosProvider, useVideos } from "./Context";
+
+export { VideosProvider, useVideos };
```
Comment (junaidshaikh-js) [inline src/context/videos/index.js:3]: Why not export in a single line? 

`export { VideosProvider, useVideos } from './Context';`

**Review by junaidshaikh-js — APPROVED** [review]
Great work! 

Clean code and good folder structure. Components are well structured into smaller parts. 

Can be merged.

File: src/context/videos/Context.jsx [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/videos");
+        if (response.status === 200) {
+          setVideos(response?.data?.videos);
+        }
+      } catch (e) {
+        console.error(e);
+      }
```
Comment (raman1999) [inline src/context/videos/Context.jsx:30]: remove console, can print some toast message in error

File: src/pages/explore/VideoListing/VideoCard/VideoCard.module.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  /* justify-content: space-between; */
+  /* height: fit-content; */
+  background-color: var(--gray3);
```
Comment (raman1999) [inline src/pages/explore/VideoListing/VideoCard/VideoCard.module.css:6]: remove this comments

**Review by raman1999 — APPROVED** [review]
Code looks clean to me,  liked the way you structured your folder, suggested some minor changes, rest looks good

File: src/context/videos/index.js (hunk 1 above)
Comment (ttahm3d) [reply src/context/videos/index.js:3]: Not sure. I like to keep things separated.

File: src/context/videos/Context.jsx (hunk 2 above)
Comment (ttahm3d) [reply src/context/videos/Context.jsx:30]: Not final version will fix.

File: src/pages/explore/VideoListing/VideoCard/VideoCard.module.css (hunk 3 above)
Comment (ttahm3d) [reply src/pages/explore/VideoListing/VideoCard/VideoCard.module.css:6]: I need them for future reference. Will remove once done with project

**Pull request merged**